storage/system_storage/SystemStorage.cpp

```diff
@@ -13,35 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "features/storage/blockdevice/BlockDevice.h"
-#include "features/storage/filesystem/FileSystem.h"
-#include "features/storage/filesystem/fat/FATFileSystem.h"
-#include "features/storage/filesystem/littlefs/LittleFileSystem.h"
+#include "blockdevice/BlockDevice.h"
+#include "filesystem/FileSystem.h"
+#include "fat/FATFileSystem.h"
+#include "littlefs/LittleFileSystem.h"
```

Comment:
Something I'd like to discuss:
In terms of user experience, including "**filesystem**/FATFileSystem.h", "**filesystem**/LittleFileSystem.h" in code seems more sensible than "**fat**/FATFileSystem.h", "**littlefs**/LittleFileSystem.h". If we want to achieve that, we can have
```
storage
├── filesystem
│   ├── fat
│   │   └── include
│   │       └── filesystem   # changed from fat
│   │           └── FATFileSystem.h
```
Of course, this would be a slight change from our initial proposal. What do you think? @rajkan01 @evedon

Reply:
It breaks the consistency in the directory structure for little benefit. I don't think it is worth doing.

Reply:
Okay